desktop/cmp/grid/impl/filter/custom/CustomRow.ts

```diff
@@ -82,7 +82,9 @@ const inputField = hoistCmp.factory<CustomRowModel>(({model}) => {
     } else if (fieldSpec.isDateBasedFieldType) {
         return dateInput({
             ...props,
-            valueType: fieldSpec.fieldType as 'localDate' | 'date'
+            valueType: fieldSpec.isMismatchedDateFieldType
+                ? 'date'
+                : (fieldSpec.fieldType as 'localDate' | 'date')
```

Comment:
I noticed that if you picked a date and then closed and reopened the popover, the dateInput wouldn't render the value, due to the difference in fieldTypes.

Reply:
Makes sense - we need to input to be able to read `dates` since that is what the filter is actually using 👍